### DISCUSSION TITLE
Welcome to CrownEd Discussions!

### DISCUSSION THREAD

**Discussion by Ajwebdevs** [AJswebdevs/CrownEd#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jais001-sushant** [comment]
Sushant this side ~
Working as a Backend Developer for this Project.

**Comment by ChhaviAmbasta** [comment]
Chhavi this side-
Working as a Frontend Developer for this project

**Comment by chehak-jindal** [comment]
Chehak Jindal this side ~ 
Working on the database for this project

**Comment by Naman6115** [comment]
Naman this side -
Working as a Backend Developer for this project
